jaeger-core/src/main/java/com/uber/jaeger/Tracer.java

```diff
@@ -21,6 +21,17 @@
  */
 package com.uber.jaeger;
 
+import java.io.InputStream;
```

Comment:
I prefer that you don't reorder diffs in this commit. If it's difficult to undo, we can let it be and address later with #147